demos/python_demos/object_detection_demo_faceboxes/object_detection_demo_faceboxes.py

```diff
@@ -0,0 +1,132 @@
+    args.add_argument("-i", "--input", type=str, nargs='+', default='', help="path to video or image/images")
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
+                           "acceptable. The demo will look for a suitable plugin for device specified. "
+                           "Default value is CPU", default="CPU", type=str)
+    args.add_argument("--labels", help="Optional. Path to labels mapping file", default=None, type=str)
+    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+    args.add_argument("-u", "--utilization_monitors", default="", type=str,
+                      help="Optional. List of monitors to show initially.")
+
+    return parser
+
+class ImageReader(object):
+    def __init__(self, file_names):
+        self.file_names = file_names
+        self.max_idx = len(file_names)
+
+    def __iter__(self):
+        self.idx = 0
+        return self
+
+    def __next__(self):
+        if self.idx == self.max_idx:
+            raise StopIteration
+        img = cv2.imread(self.file_names[self.idx], cv2.IMREAD_COLOR)
+        if img.size == 0:
+            raise IOError('Image {} cannot be read'.format(self.file_names[self.idx]))
+        self.idx += 1
+        return img
+
+
+class VideoReader(object):
+    def __init__(self, file_name):
+        try:
+            self.file_name = int(file_name[0])
+        except:
+            self.file_name = file_name[0]
+
+
+    def __iter__(self):
+        self.cap = cv2.VideoCapture(self.file_name)
+        if not self.cap.isOpened():
+            raise IOError('Video {} cannot be opened'.format(self.file_name))
+        return self
+
+    def __next__(self):
+        was_read, img = self.cap.read()
+        if not was_read:
+            raise StopIteration
+        return img
+
+def main():
+    args = build_argparser().parse_args()
+
+    ie = IECore()
+    detector = Detector(ie, args.model, args.device)
+
+    img = cv2.imread(args.input[0], cv2.IMREAD_COLOR)
+    frames_reader, delay = (VideoReader(args.input), 1) if img is None else (ImageReader(args.input), 0)
+
+    if args.labels:
+        with open(args.labels, 'r') as f:
+            labels_map = [x.strip() for x in f]
+    else:
+        labels_map = None
+
+    presenter = monitors.Presenter(args.utilization_monitors, 25)
+    for frame in frames_reader:
+        detections = detector.detect(frame)
+        presenter.drawGraphs(frame)
+        for x_min, x_max, y_min, y_max, score, label in zip(detections['x_mins'], detections['x_maxs'],
+                                                     detections['y_mins'], detections['y_maxs'],
+                                                     detections['scores'], detections['labels']):
+            x_min = max(0, x_min).astype(np.int)
+            y_min = max(0, y_min).astype(np.int)
```

Comment:
If `y_min` is negative it fails with `'int' object has no attribute 'astype'`

Reply:
Replaced astype with int()